pkg/ccl/streamingccl/streamingest/stream_ingestion_dist.go

```diff
@@ -55,6 +62,18 @@ var replanFrequency = settings.RegisterDurationSetting(
 	settings.PositiveDuration,
 )
 
+var persistExecutionDetailsFrequency = settings.RegisterDurationSetting(
```

Comment:
I'd prefer not to add a cluster setting for this, but rather just leave it as a const or env var for now or make it an option on the job.

If we must use a cluster setting, it would only be for the system tenant, right?

Reply:
changed it to a const = 10 * time.Minute for now. This will go away once we have the resumer specific method.